umh-core/pkg/gatekeeper/certificatehandler/handler.go

```
@@ -0,0 +1,347 @@
+	h.setCertificate(strings.ToLower(resp.UserEmail), cert, chain)
+
+	h.mu.RLock()
+	encryptedRootCA := h.encryptedRootCA
+	authToken := h.authToken
+	instanceUUID := h.instanceUUID
+	h.mu.RUnlock()
+
+	if resp.RootCA != "" && resp.RootCA != encryptedRootCA {
+		h.setEncryptedRootCA(resp.RootCA)
+
+		hasher := sha3.New256()
```

golangci-lint (2.12.2)

[error] inline: Call of sha3.New256 should be inlined

(govet)